packages/cli/src/components/ProjectStatus/ServerProjectStatus.tsx

```diff
@@ -95,7 +94,9 @@ export default function ServerProjectStatus({
         break
 
       case "BUILD_RPCS_SUCCESS":
-        setStatus("Server is ready, waiting for changes.")
+        setStatus(
+          `Server is ready, waiting for changes: http://localhost:${command.port}`,
```

Comment:
```suggestion
          `Server is ready, listening on port ${command.port} for changes`,
```
I think we're not really binding on `localhost`, are we? Then instead I'd prefer to say this. It might be confusing otherwise.

Reply:
Ah, better! It is listening for both changes and requests though, so I'll take out the last part ;)